src/Sulu/Bundle/TrashBundle/Resources/js/containers/RestoreFormOverlay/RestoreFormOverlay.js

```diff
@@ -58,8 +59,15 @@ class RestoreFormOverlay extends React.Component<Props> {
             return;
         }
 
-        const formStore = memoryFormStoreFactory.createFromFormKey(formKey);
-        formStore.loading = true;
+        const formStore = new SchemaFormStoreDecorator(
```

Comment:
setting `store.loading` to `true` in the initializer function prevents a race condition that leads to an infinite loader in the overlay